client/src/tests/Login.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
Add test when login button is clicked for non-web mode

Reply:
Hmm, I haven't figured out a way to mock microapp API calls from our testing framework yet, will resolve Microapp-specific UI/Action Tests after #88

Reply:
Alright